files/matreshka/update.json

```diff
@@ -3,6 +3,13 @@
   "name": "matreshka",
   "repo": "matreshkajs/matreshka",
   "files": {
-    "include": ["matreshka.min.js", "matreshka.js", "matreshka.min.map"]
+    "include": [
+		"matreshka.min.js",
+		"matreshka.js",
+		"matreshka.min.map",
+		"magic/matreshka-magic.min.js",
```

Comment:
I'd like to get another folder. But it's up to you, make it as easy as needed. I'm OK if the files will be placed in one folder.